.gitignore

```diff
@@ -5,6 +5,7 @@
 ### editor leavings ###
 *~
 \#*
+.vscode
```

Comment:
I don't mind this change, but just so you know, you can edit `.git/info/exclude` to make .gitignore rules that don't apply to everybody.  See:  https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally

Reply:
Awesome. I like it.